spec/ddtrace/contrib/rails/support/rails3.rb

```diff
@@ -113,6 +114,8 @@ def draw_test_routes!(mapper)
   # Version of Ruby < 4 have initializers with persistent side effects:
   # actionpack-3.0.20/lib/action_view/railtie.rb:22
   def after_rails_application_creation
+    Lograge.remove_existing_log_subscriptions if Object.const_defined?('Lograge')
```

Comment:
```suggestion
    Lograge.remove_existing_log_subscriptions if defined? ::Lograge
```

Does this work the same way?